Source/WebCore/Modules/fetch/FetchRequest.cpp

```diff
@@ -245,8 +245,11 @@ ExceptionOr<void> FetchRequest::initializeWith(FetchRequest& input, Init&& init)
         auto fillResult = init.headers ? m_headers->fill(*init.headers) : m_headers->fill(input.headers());
         if (fillResult.hasException())
             return fillResult;
-    } else
+        m_navigationPreloadIdentifier = { };
```

Comment:
Is this line needed?

Reply:
Yes, we still need to not use preload if headers ever gets changed.